libpod/container.go

```diff
@@ -718,6 +728,21 @@ func (c *Container) Export(path string) error {
 	return err
 }
 
+// AddArtifacts creates and writes to an artifact file for the container
+func (c *Container) AddArtifacts(name string, data []byte) error {
+	return ioutil.WriteFile(filepath.Join(c.config.StaticDir, "artifacts", name), data, 0740)
+}
+
+// GetArtifacts reads the specified artifact file from the container
+func (c *Container) GetArtifacts(name string) ([]byte, error) {
+	return ioutil.ReadFile(filepath.Join(c.config.StaticDir, "artifacts", name))
```

Comment:
Does this return a good error when the file doesn't exist? If it doesn't can we do an errors.Wrapf to give more details

Reply:
the error would be something along the lines of "file not found" and all. I wasn't sure whether to wrap it or not

Reply:
We can also always wrap the error where GetArtifact() would be called.

Reply:
File not found is probably sufficient, though it might be helpful to do a Wrapf so we can throw the container ID and artifact name in there ("error locating artifact %s in container %s" or something)